common/app/views/fragments/immersiveGalleryHeadTonal.scala.html

```diff
@@ -0,0 +1,90 @@
+@(item: model.ContentType, showBadge: Boolean = false, showMeta: Boolean = true)(implicit request: RequestHeader)
+
+@import views.support.TrailCssClasses.toneClass
+@import views.support.ContributorLinks
+
+<header class="content__head content__head--immersive content__head--desktop is-fixed-height tonal__head tonal__head--@toneClass(item)">
```

Comment:
on the topic of common, we should really only keep things that are genuinely common in here not just because stuff is needed in more than one project.  We have a lot of stuff that's only common between two or more of facia/applications/article which needn't be compiled into admin/diagnostics/etc.  I wonder if it would be better to start putting things in a new module.. I've been wondering for a while though 💭